build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
Given the fact that `ToStatement` is [documented](https://www.playframework.com/documentation/2.5.x/ScalaAnorm#Custom-parameter-conversions) as a feature, it's very likely this is being used in other libraries, so binary compatibility issues are likely. The same is true for `Column`.

We could work around this by defining a separate implicit class for the ops:
```scala
implicit class ToStatementOps[A](self: ToStatement[A]) {
  final def contramap[B](f: B => A): ToStatement[B] = ToStatement[B] { (s, i, b) => self.set(s, i, f(b)) }
}
```